third_party/terraform/tests/resource_healthcare_dataset_test.go.erb

```diff
@@ -106,33 +106,6 @@ func TestAccHealthcareDataset_basic(t *testing.T) {
 	})
 }
 
```

Comment:
These test files can be swapped from being erb's to straight `.go` files. They are only templated files so they can ben excluded from the GA provider.

Reply:
Done.